Add changesets for DAM scoping (PR #976) (#1397)
vivid-planet/comet: .changeset/ten-numbers-sip.md

---
"@comet/cms-admin": major
"@comet/cms-api": major
---

Add scoping to the DAM

The DAM scoping can be enabled optionally. You can still use the DAM without scoping.

To enable DAM scoping, you must

- In the API: Pass a `Scope` class to the `DamModule`

> - In the admin app: Add a `DamConfigProvider` with `scopeParts` (e.g. `<DamConfigProvider value={{ scopeParts: ["domain"] }}>`)

Reviewer: ```suggestion
- In the admin app: Set `scopeParts` in the `DamConfigProvider` (e.g. `<DamConfigProvider value={{ scopeParts: ["domain"] }}>`)
```

The DamConfigProvider was already present before

> You can access the current DAM scope in the admin app using the `useDamScope()` hook.

Author: @johnnyomair Maybe I missed something. Feel free to extend this
